linalg/ode.cpp

```diff
@@ -446,6 +470,7 @@ AdamsMoultonSolver::AdamsMoultonSolver(int s_, const double *a_)
    smax = std::min(s_+1,5);
    a = a_;
    k = new Vector[5];
```

Comment:
Why make a pointer of size 5 rather than smax?

Reply:
You are right. This should be smax. Note:
1)  This is not part of the PR.
2)  This issue is fixed in #3480